big_vision/configs/proj/distill/bigsweep_pets.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Big Vision Authors.
```

Comment:
This file is here by mistake. It doesn't exist anymore internally, so this makes me think that maybe the PR is based on a somewhat stale version? Could you double-check?

Reply:
ah right ... the `rsync` was missing the `--del` flag; thanks for the catch!